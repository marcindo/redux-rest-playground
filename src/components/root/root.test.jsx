
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Root from '.';

it('Renders root', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<Root />);
  const component = renderer.getRenderOutput();

  expect(component).toMatchSnapshot();
});
