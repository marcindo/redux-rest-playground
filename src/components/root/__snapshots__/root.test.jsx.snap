// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders root 1`] = `
<div
  className="root"
>
  <styled.input />
  <Connect(Connect(PostsList)) />
</div>
`;
